test/unit_test/deploy/test_migration_volumes.py

```
@@ -0,0 +1,182 @@
+    """Run ``migration.sh help`` and return stdout.
+
+    The script's ``main()`` calls ``check_docker`` before any
+    operation-dispatch, so this test requires Docker to be available.
+    Skip if not -- the per-engine mapping tests above already cover the
+    helper function in isolation.
+    """
+    if not MIGRATION_SH.is_file():
+        pytest.fail(f"missing script: {MIGRATION_SH}")
+    if shutil.which("bash") is None:
+        pytest.skip("bash is not available on this platform")
+    if shutil.which("docker") is None or subprocess.run(["docker", "info"], capture_output=True, text=True, check=False).returncode != 0:
```

Ruff (0.16.1)

[error] Starting a process with a partial executable path

(S607)